particula/aerosol_dynamics/parcel.py

```diff
@@ -1,45 +1,35 @@
 """
 Class for creating a air parcel to put particles and gases in.
 """
-
-from particula.aerosol_dynamics import environment, particle
+import numpy as np
+from particula.aerosol_dynamics import particle, u
 
 
 class Parcel:
-    """Sets the class for creating a parcel.
+    """
+    Sets the class for creating a parcel.
     This starts the base for particle and gas dynamic simulations.
     Like the eveolution of a size distribution with time.
     Or change in temperature causing particle evaporation or gas condensation.
 
     Attributes:
-        name            (str)           The name of the parcel.
-        particle_data   (list)          The radius of the particle.
-        enviroment      (Environment)   The enviroment of the parcel.
-        TODO: after enviromnetal class is added to main.
-        temperature (float) default: The air temperature in Kelvin.
-        pressure (float) default: The air pressure in atm?.
-        add gases class
+        name            (str)               The name of the parcel.
+        particle_data   (object list)       Particle objects.
```

Comment:
I think `objects` is confusing here. Can you be more specific? "particle list"?

Reply:
sure :)